src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_quantization.cpp

```diff
@@ -713,9 +713,6 @@ void prepare_quantization::prepare_asymmetric_quantization(program &p, convoluti
 
     // Remove sub operations from the graph and set correct users for zero points and inputs
     if (asymmetric_data) {
-        if (!new_a_zp || !new_input)
-            CLDNN_ERROR_MESSAGE(new_conv_node.id(), "Unexpected nullptr in asymmetric quantization for activations optimization");
```

Comment:
why is it removed instead of changing to `OPENVINO_ASSERT`?

Reply:
Because this will never be null in case of asymmetric_data